tests/test_qagent.py

```diff
@@ -0,0 +1,11 @@
+from metagamer.environments import qagent
+
+
+def test_qagent_runs():
```

Comment:
This test is too basic imo. It checks everything works when put together, but not that the units actually do anything useful.

Reply:
Yep, agreed.  Only better than not test at all, which was the only other alternative in the time I had ;-) But let's focus on your improved abstraction.